### ISSUE TITLE
Bodymovin doesn't exports my animations

### ISSUE THREAD

**Issue by ThePix3l** [airbnb/lottie-web#2213]
Hi guys,

I'm new with creating lotties. 
I have an animation I have used as move before and would like to use it as json now.
It's a very simple animation but bodymovin stops rendering in the next moment after i push the render button and doesn't go on anymore. 
I don't know why because there are no error informations.

Do you have some idea?

After Effects 15.1.2 (Build 69)

![bodymovin](https://user-images.githubusercontent.com/66958974/84671702-f9ef0d00-af27-11ea-9f77-37e4c796b9ac.jpg)

**Comment by ThePix3l** [comment]
Ok,
I think I know why...
It's because I have used Illustrator files in my animations.

**Comment by maloriaga** [comment]
I have the same issue. Does anybody know if is this the reason? (using Ai files)

**Comment by bodymovin** [comment]
Yes, there are issues with older versions of AE and bodymovin v5.6.x
I suggest to upgrade AE or use an older version, like any v5.5.x

**Comment by ThePix3l** [comment]
It works when you convert the Illustrator layers into shape layers in a composition in AE.
1. Select the Illustrator layer to convert.
2. Go to Layer > Create > Create Shapes from Vector Layer.
3. Open and adjust the layer options for the newly created shape layer to take advantage of the unique animation possibilities provided by shape layers.
